.travis.yml

```diff
@@ -42,7 +42,8 @@ notifications:
     - joshua.suereth@typesafe.com
 
 script:
-  - sbt -J-XX:ReservedCodeCacheSize=128m "$SCRIPTED_TEST"
+  - sbt ";publishLocal;project bundledLauncherProj;publishLocal;project precompiled-2_9_2;publishLocal;project precompiled-2_8_2;publishLocal;project precompiled-2_9_3;publishLocal"     
+  - sbt -J-XX:ReservedCodeCacheSize=128m -Dsbt.version=0.13.9-SNAPSHOT "$SCRIPTED_TEST"
```

Comment:
The new process need to use sbt in version 0.13.9-SNAPSHOT. If we do not define the version - it will be red from `project/build.properties`. But there is 0.13.8. And we come back to the "NoSuchMethod" problem.